pack.yaml

```diff
@@ -3,7 +3,7 @@
 ref: mqtt
 name: mqtt
 description: MQTT Integration for StackStorm
-version: 1.0.1
+version: 1.0.2
```

Comment:
How about 1.1.0? These changes seem like a bit more than just a bugfix.

Reply:
@cognifloyd sure, makes sense to me! i'll push that change